src/chigame/games/tournaments_function.py

```diff
@@ -0,0 +1,132 @@
+"""
+This file contains the functions needed for the tournaments feature.
+"""
+
+import random
+
+from .models import Lobby, Match, Player, Tournament
+
+
+def create_tournaments_brackets(tournament: Tournament) -> list[Match]:
+    """
+    Creates a list of brackets for the tournaments.
+
+    Args:
+        tournament: the tournament
+
+    Returns:
+        a list of matches
+    """
+    players = [player for player in tournament.players.all()]  # the players in the tournament
+    brackets = []
+    random.shuffle(players)  # shuffle the players
+    # Create a list of brackets (match assignment) for the tournament
+    for i in range(0, len(players), tournament.game.max_players):
+        game = tournament.game
+        lobby = Lobby.objects.create(
+            match_status=Lobby.Lobbied,
+            game=game,
+            game_mod_status=Lobby.Default_game,
+            created_by=players[i],  # the field is currently set to the first player in the list
+            min_players=game.min_players,
+            max_players=game.max_players,
+            # time_constraint, # default: 300
+            # lobby_created, # default: the field is set to the current time
+        )
+        players_in_match = players[i : i + tournament.game.max_players]  # the players in the match
```

Comment:
Seems like this line is adding all the players into the match. Shouldn't it just be one additional player besides the one in the loop? When I create the tournament, I get this: 
<img width="668" alt="image" src="https://github.com/uchicago-cs/chigame/assets/102563648/0c7bb234-c8cb-4b5b-9be2-a131d1917fcf">
You could fix this by incrementing through the players list by 2 and creating matches that way. 
`for i in range(0,len(players),2)` and then `players_in_match = players[i:i+1]`.